src/index.css

```diff
@@ -11,7 +11,8 @@ body {
     'Fira Sans',
     'Droid Sans',
     'Helvetica Neue',
-    sans-serif;
+    sans-serif,
+    RobotoSlab;
```

Comment:
- [ ] The current way in which you have added the `font-family` property is a top-down approach, where if the first font is available, it is applied throughout the file, if not the next font is applied if available, and so on. This means the whole app has one font family, but this is not the case with the given design. We see that some elements have the `Montserrat` font while others have the `RobotoSlab`. Kindly update your styles to match the given design

Given design
![image](https://github.com/RileyManda/bookstore/assets/81932478/a05e343b-ee05-4725-abae-403d2fdb14e6)

Your design
![image](https://github.com/RileyManda/bookstore/assets/81932478/79eb50ec-7f50-42ae-bbbb-6aca1759cfcd)

- The given design uses `RobotoSlab` for all book card components except the `Completed` element which uses `Montserrat`, kindly fix this.

### Kindly use reusable classes for best practices.

Reply:
Noted with thanks @Munsa1 🙏 .I will work on this now.